executor/show_test.go

```diff
@@ -609,6 +609,19 @@ func (s *testSuite5) TestShowCreateTable(c *C) {
 			"  `a` varchar(123) DEFAULT NULL\n"+
 			") ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin",
 	))
+
+	// for expression index
```

Comment:
Should also check out  `select * from information_schema.tidb_indexes` showing probably results.

Reply:
Great, I will let this support expression in following PR